sda-orch/values.yaml

```diff
@@ -0,0 +1,70 @@
+
+image:
+  repository: neicnordic/sda-orch
+  tag: latest
+  pullPolicy: Always
+
+# Annotations applied to pods of all services.
+# This should be a multi-line string mapping directly to the a map of
+# the annotations to apply to the service pods
+podAnnotations: {}
+
+# If an external PKI infrastructure is used set this to true
+pkiService: false
+# if permissions needs to be set on the injected certificates set this to true
+pkiPermissions: true
+
+# Assume RBAC is enabled in the cluster
+rbacEnabled: true
+
+podSecurityPolicy:
+  create: true
+
+networkPolicy:
+  create: false
+
+tlsPath: "/tls/certs"
+
+## RevisionHistory
+##  If defined, set the revisionHistoryLimit of the deployment, defaults to 3
+##  RevisionHistoryLimit is number of old ReplicaSets to retain to allow rollback.
+  # revisionHistory: 3
+
+# If an external secrets management service is used set this to true
+secretsService: false
+
+# used by both Message Broker and Database
+sslmode:
+  ssl: true
+  verifyPeer: true
+
+db:
+  host: ""
+  name: "lega"
+  userIngest: ""
+  userOutgest: ""
+  passIngest: ""
+  passOutgest: ""
+  port: 5432
+  sslMode: "verify-full"
+
+broker:
+  exchange: ""
+  host: ""
+  password: ""
+  port: 5671
+  username: ""
+  vhost: "/"
+  queue:
+    inbox: ""
+    completed: ""
+    verify: ""
+
+
+resources:
+  requests:
+    memory: "128Mi"
+    cpu: "100m"
```

Comment:
Is it really this power hungry?

Reply:
not really, i think could be lower, there are just 3 processes that pass info around. I will update.

Reply:
this seems to be the limit on other services as well ... I would rather keep it consistent